terraform/prod.tfvars

```diff
@@ -42,6 +42,8 @@ tenant_cidrs = [
   "52.213.138.76/32",
   "52.213.151.67/32",
   "52.208.47.129/32",
+  #  Cabinet office trial
```

Comment:
Copy-paste left over. Also I corrected two space mistake in the commit message. Now we've got perfect two line change.